Gemfile

```diff
@@ -68,5 +75,12 @@ group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
   gem "selenium-webdriver"
-
+  gem "simplecov"
 end
+
+gem "devise"
```

Comment:
better move them to the top, before all the groups

Reply:
Fixed.